src/lib/form/yup.custom.ts

```diff
@@ -52,7 +52,7 @@ yup.addMethod<yup.StringSchema>(
 );
 
 type MaxAmountValidationParams = {
-  maxAmount: MonetaryAmount<CurrencyExt> | Big;
+  maxAmount: MonetaryAmount<CurrencyExt> | Big | undefined;
```

Comment:
Not quite sure if this is what you meant by allowing maxAmount to be undefined. Looks to be handled differently in the swap, so there may be a better way of doing this.

Reply:
I don't think there should be a need to change our yup validations, but correct me if Im wrong

Reply:
Do you remember we talked about this? Because the value is undefined on load, it throws an error.